pkgs/tools/security/certmgr/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  version = "1.6.1";
+  name = "certmgr-${version}";
+
+  goPackagePath = "github.com/cloudflare/certmgr/";
+
+  nativeBuildInputs = [ ];
```

Comment:
Can be removed

Reply:
Of course. Though I already did that. Apparently not.. Removed now :)